src/PowerShellEditorServices/Services/DebugAdapter/Handlers/DisconnectHandler.cs

```diff
@@ -75,7 +75,7 @@ public async Task<DisconnectResponse> Handle(DisconnectArguments request, Cancel
 
 #pragma warning disable CS4014
             // Trigger the clean up of the debugger. No need to wait for it.
-            Task.Run(_psesDebugServer.OnSessionEnded);
+            Task.Run(_psesDebugServer.OnSessionEnded, cancellationToken);
```

Comment:
@rjmholt Should this one be explicitly `CancellationToken.None`? Presumably we probably don't want to the debug server's clean up to be cancelled.

Reply:
But that said if it's cancelled here then we don't return the `DisconnectResponse` so I suppose it would get cancelled again?

Reply:
Yeah I think `None` seems reasonable

Reply:
Oops forgot about this 🤷